src/transformers/models/convbert/param_mapping.py

```diff
@@ -0,0 +1,102 @@
+# coding=utf-8
```

Comment:
not a big fan of this file - we usually don't have such a file for models. I think I'd prefer to just have this function inside of `load_tf_weights_in_convbert`, but not too big of a deal for me...